extra/1-currency-conversion.js

```diff
@@ -5,7 +5,10 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD(Pound) {
+  return Pound*1.4;
```

Comment:
Nice! Don't forget spaces to improve readability, though :)

Reply:
> Nice! Don't forget spaces to improve readability, though :)

Thank you. i have added space now.

> Hi Junita, really great job! Lots of clear coding in here and you've clearly picked up the concepts well. You went the extra mile with the exercises and have really mastered string interpolation which is great to see. Watch out for typos as they can make life difficult when you're on the job :) (the amount of hours I've spent sifting through my code because of a silly typo...).

THANK YOU so much for your reviews. It is really helpful for me to realise my mistakes. I will use and remember all the suggestions you have given me here in future too .